services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/outputs/_context.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass, field
+from pathlib import Path
+
+import aioprocessing
+from aioprocessing.queues import AioQueue
+from fastapi import FastAPI
+
+from ..mounted_fs import MountedVolumes
+
+
+@dataclass
+class OutputsContext:
+    outputs_path: Path
+
+    # _PortKeysEventHandler (generates) -> EventFilter (receives)
+    port_key_events_queue: AioQueue = field(default_factory=aioprocessing.AioQueue)
```

Comment:
Yes, gladly.
To use a "generic queue" with a Process, you cannot just use `asyncio.Queue` nor `queue.Queue` (aka threading queue), you need to use the proper one form `multiprocessing`.

Unfortunately it's pretty tricky, since it does not have support for asyncio. asyncprocessing builds on top of the standard library and adds methods that can be awaited.